src/main/java/org/example/api/DeliveryEmployeeService.java

```diff
@@ -35,6 +36,23 @@ public int createDeliveryEmployee(DeliveryEmployeeRequest employee) throws Proje
             return id;
 
     }
+
+    public void updateDeliveryEmployee(int id, DeliveryEmployeeRequest deliveryEmployee) throws ProjectException, SQLException, DeliveryEmployeeDoesNotExistException {
+        String validation = deliveryEmployeeValidator.isValidUpdate(deliveryEmployee);
+
+        if (validation != null) {
+            throw new ProjectException();
```

Comment:
could we have a ValidationFailed exception that allows you to pass in what failed rather than this generic one?